frontend/src/Components/CustomServicesNav/CustomServicesNav.scss

```diff
@@ -16,16 +22,20 @@
   color: $color-gray;
   h2 {
     font-size: $font-size-subtitle;
-    margin-bottom: $default-spacing / 2;
+    @include desktop {
+      margin-bottom: $default-spacing / 2;
+    }
     color: black;
```

Comment:
can we change this to $color-black

Reply:
yep, although used `$color-primary` instead.